src/index.html

```diff
@@ -4,9 +4,304 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__links">
+        <a href="#homePage" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav>
+          <ul class="header__list">
+            <li class="header__link">
+              <a href="#Apple" class="header__link-a is-active">APPLE</a>
```

Comment:
```suggestion
              <a href="#Apple" class="header__link-a is-active">Apple</a>
```
Лучше задать uppercase в стилях